src/chi-vue/src/views/DataTableView.vue

```diff
@@ -0,0 +1,1086 @@
+              }
+            ]
+          },
+          {
+            id: 'NTM000021097',
+            active: false,
+            data: [
+              {},
+              { template: 'ticketId', payload: { id: 'NTM000021097' } },
+              { template: 'status', payload: { status: 'active' } },
+              'Colocation',
+              'Internet Advantage',
+              '04/05/2018 8:00 AM',
+              'SVUJW034781A',
+              {
+                template: 'actions',
+                payload: { id: 'NTM000021097' },
+                align: 'right'
+              }
+            ]
+          },
+          {
+            id: 'NTM000021098',
+            active: false,
+            data: [
+              {},
+              { template: 'ticketId', payload: { id: 'NTM000021098' } },
+              { template: 'status', payload: { status: 'inactive' } },
+              'Colocation',
+              'Internet Advantage',
+              '04/05/2018 8:00 AM',
+              'SVUJW034781A',
+              {
+                template: 'actions',
+                payload: { id: 'NTM000021098' },
+                align: 'right'
+              }
+            ]
+          },
+          {
+            id: 'NTM000021099',
+            active: false,
+            data: [
+              {},
+              { template: 'ticketId', payload: { id: 'NTM000021099' } },
+              { template: 'status', payload: { status: 'inactive' } },
+              'Colocation',
+              'Internet Advantage',
+              '04/05/2018 8:00 AM',
+              'SVUJW034781A',
+              {
+                template: 'actions',
+                payload: { id: 'NTM000021099' },
+                align: 'right'
+              }
+            ]
+          },
+          {
+            id: 'NTM000021100',
+            active: false,
+            data: [
+              {},
+              { template: 'ticketId', payload: { id: 'NTM000021100' } },
+              { template: 'status', payload: { status: 'active' } },
+              'Colocation',
+              'Internet Advantage',
+              '04/05/2018 8:00 AM',
+              'SVUJW034781A',
+              {
+                template: 'actions',
+                payload: { id: 'NTM000021100' },
+                align: 'right'
+              }
+            ]
+          },
+          {
+            id: 'NTM000021101',
+            active: false,
+            data: [
+              {},
+              { template: 'ticketId', payload: { id: 'NTM000021101' } },
+              { template: 'status', payload: { status: 'inactive' } },
+              'Colocation',
+              'Internet Advantage',
+              '04/05/2018 8:00 AM',
+              'SVUJW034781A',
+              {
+                template: 'actions',
+                payload: { id: 'NTM000021101' },
+                align: 'right'
+              }
+            ]
+          },
+          {
+            id: 'NTM000021101',
```

Comment:
Doesn't this have another id?
```suggestion
            id: 'NTM000021102',
```

Reply:
Good catch!